lib/kernel-manager.coffee

```diff
@@ -77,6 +77,14 @@ class KernelManager
                 )
                 finishKernelStartup kernel
 
+    attachKernel: (grammar, kernel, onAttach) ->
```

Comment:
As it stands there is no need for the `onAttach` callback (because this method isn't asynchronous).